<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Accumulate</class>
 <widget class="QWidget" name="Accumulate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1046</width>
    <height>806</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Accumulate</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QToolButton" name="tBtnAccumulateEnable">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Enable Accumulate</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelAccStatus">
       <property name="text">
        <string>status</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Noise</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0">
      <item>
       <layout class="QGridLayout" name="gLayNoise"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="hLayNoiseOption">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="hLaySmooth">
          <item>
           <widget class="QLabel" name="labelSmoothWindow">
            <property name="text">
             <string>smooth window</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBoxSmoothWindow">
            <property name="maximum">
             <number>999</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="hLayPolyOrder">
          <item>
           <widget class="QLabel" name="labelPolyOrder">
            <property name="text">
             <string>polynomial order</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBoxPolyOrder"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="vLayCount">
          <item>
           <layout class="QHBoxLayout" name="hLayNoiseCount">
            <item>
             <widget class="QLabel" name="labelNoiseCount">
              <property name="text">
               <string>count</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBoxNoseCount">
              <property name="maximum">
               <number>999</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="tBtnNoiseEnable">
              <property name="text">
               <string>enable</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="labelCountStatus">
            <property name="text">
             <string>status</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
